pushapkscript/docker.d/worker.yml

```diff
@@ -15,6 +15,68 @@ taskcluster_scope_prefixes:
 products:
   $flatten:
     $match:
+      'COT_PRODUCT == "firefox" && ENV == "prod"':
+        - product_names: [ "fenix" ]
+          digest_algorithm: 'SHA-256'
+          skip_check_multiple_locales: true
+          skip_check_same_locales: true
+          skip_checks_fennec: true
+          apps:
+            fenix-nightly:
+              package_names: [ "org.mozilla.fenix" ]
+              certificate_alias: 'fenix-nightly'
+              google:
+                default_track: 'production'
+                credentials_file: { "$eval": "GOOGLE_CREDENTIALS_FENIX_NIGHTLY_PATH" }
+            fenix-aab-nightly:
```

Comment:
Oops! 'fenix-aab-nightly' should not be in use; it was used for our initial AAB testing of Fenix Nightly. I should have cleaned that up - sorry. Anyway, no need to copy it.

Reply:
Thanks, removed.